src/website/src/pages/ec/components/message/docs/usage.mdx

```diff
@@ -14,7 +14,7 @@ import { Paragraph, Anatomy, Do, Caution } from '@ecl/website-components';
 ## Anatomy
 
 <Anatomy
-  image="https://inno-ecl.s3.amazonaws.com/media/images/EC/Messages/Message.jpg"
+  image="https://inno-ecl.s3.amazonaws.com/media/images/EC/Messages/Message2.jpg"
```

Comment:
the image for mobile version is missing (even if it close, this is not the same)